buddyboss/buddyboss-platform
# Fix for showing empty space at the top of link preview and link is not hide when there is additional text

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2442 #866 

### How to test the changes in this Pull Request:

1. Add link in the activity post area and create a post with link preview. You should not see any empty space at the top.
2. Create another activity post - this time add a link then add some additional text. The link should be hide from the activity post.
3. Youtube, vimeo video link will also work this way.
4. If there is multiple link added in the activity post then all links will be visible.

### Proof Screenshots or Video
https://www.loom.com/share/6d8cabc992f24cd08ef8ed323fc51037


### Other information:




### Changelog entry

> Removed empty space at the top of link preview.
> Improved link preview functionality
